internal/user/repository/eventsourcing/eventstore.go

```diff
@@ -1438,6 +1303,217 @@ func (es *UserEventstore) verifyMfaOTP(otp *usr_model.OTP, code string) error {
 	return nil
 }
 
+func (es *UserEventstore) AddU2F(ctx context.Context, userID string) (*usr_model.WebAuthNToken, error) {
+	user, err := es.HumanByID(ctx, userID)
+	if err != nil {
+		return nil, err
+	}
+	webAuthN, err := es.webauthn.BeginRegistration(user, usr_model.AuthenticatorAttachmentUnspecified, usr_model.UserVerificationRequirementDiscouraged, user.U2FTokens...)
```

Comment:
@livio-a please have a look at this parameters. i'm not sure if these are all correct

Reply:
at the moment we don't restrict the authenticator type and user verifications should always be discouraged for 2fa